docs/guides/nextjs.md

```diff
@@ -194,16 +196,18 @@ export const createCounterStore = (
 'use client'
 
 import { type ReactNode, createContext, useRef, useContext } from 'react'
-import { type StoreApi, useStore } from 'zustand'
+import { useStore } from 'zustand'
 
 import {
   type CounterStore,
   createCounterStore,
   initCounterStore,
 } from '@/stores/counter-store'
 
-export const CounterStoreContext = createContext<StoreApi<CounterStore> | null>(
-  null,
+export type CounterBoundStore = ReturnType<typeof createCounterStore>
```

Comment:
Can we avoid using "Bound" word? It's confusing people.
(As a side note, I think we should avoid "useBoundStore" in docs if possible.)

Reply:
@dai-shi any suggestions? Since it's StoreApi<CounterStore> what's your suggestion?

Reply:
Well, if I had an idea, I would have already changed `useBoundStore`.

Reply:
On second thought, I don't think CounterBoundStore is actually "bound".
```suggestion
export type CounterStoreApi = ReturnType<typeof createCounterStore>
```

Reply:
@dai-shi makes sense, I'll change it